rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildConfigurationEndpoint.java

```diff
@@ -152,8 +152,8 @@ public Response getAll(@ApiParam(value = PAGE_INDEX_DESCRIPTION) @QueryParam(PAG
 
     @ApiOperation(value = "Creates a new Build Configuration")
     @ApiResponses(value = {
-            @ApiResponse(code = INVLID_CODE, message = INVALID_DESCRIPTION, response = BuildConfigurationSingleton.class),
             @ApiResponse(code = SUCCESS_CODE, message = SUCCESS_DESCRIPTION, response = BuildConfigurationSingleton.class),
+            @ApiResponse(code = INVLID_CODE, message = INVALID_DESCRIPTION, response = ErrorResponseRest.class),
```

Comment:
Yeah the replace is 111 lines, should I make a new PR for this or just commit it here?